README.md

```diff
@@ -19,6 +19,19 @@ The projects the team are working on in the January 2018 firebreak are:
 **deployed prototype:** [https://registers-upvote.herokuapp.com/registers](https://registers-upvote.herokuapp.com/registers)
 
 
+## Registers Sprint 
+**A sprint-by-sprint list of what we're doing and learning that:**
+- documents achieved milestones for each sprint
+- documents user research findings
+
+**code:** [https://github.com/openregister/registers_sprint_by_sprint](https://github.com/openregister/registers_sprint_by_sprint)
```

Comment:
I'm afraid this repo does not exist so the link is 404'ing.  I am not sure what happened but neither the original repo name or this new one exists currently.

Reply:
It's 404-ing because there's no point forking the repo across until we're agreed on the name. Once this PR gets approved / merged, I'll fork again.